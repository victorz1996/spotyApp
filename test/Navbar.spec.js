import { shallowMount } from '@vue/test-utils'
import Vue from 'vue'
import Bootstrapvue from 'bootstrap-vue'
import navbar from '../components/Navbar.vue'

let wrapper
beforeEach(() => {
  Vue.use(Bootstrapvue)
  wrapper = shallowMount(navbar)
})

describe('Navbar', () => {
  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  it('HTMl renderizado structure esperada', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
