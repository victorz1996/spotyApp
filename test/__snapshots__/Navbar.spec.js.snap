// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar HTMl renderizado structure esperada 1`] = `
<div>
  <b-navbar-stub
    tag="nav"
    toggleable="sm"
    type="dark"
    variant="dark"
  >
    <b-navbar-brand-stub
      tag="div"
      target="_self"
    >
      <img
        alt="Kitten"
        class="p-1 align-top"
        height="35"
        src="/banner-ico.png"
        width="35"
      />
      SpotiApp
    </b-navbar-brand-stub>
     
    <b-navbar-toggle-stub
      label="Toggle navigation"
      target="nav-collapse"
    />
     
    <b-collapse-stub
      id="nav-collapse"
      isnav="true"
      tag="div"
    >
      <b-navbar-nav-stub
        tag="ul"
      >
        <b-nav-item-stub
          linkattrs="[object Object]"
          target="_self"
        >
          Home
        </b-nav-item-stub>
         
        <b-nav-item-stub
          linkattrs="[object Object]"
          target="_self"
        >
          Buscar
        </b-nav-item-stub>
      </b-navbar-nav-stub>
    </b-collapse-stub>
  </b-navbar-stub>
</div>
`;
